component/authenticator/src/test/java/org/wso2/carbon/identity/application/authenticator/backupcode/BackupCodeAuthenticatorTest.java

```diff
@@ -148,6 +148,8 @@ public class BackupCodeAuthenticatorTest extends PowerMockTestCase {
     private static final String HASHED_BACKUP_CODES =
             "2d578fa2a67a4e24933164a78752f9ea60cdbcbcb683637b582595e49d19a305,2dc0269fa54d269a87536810ec453cb095b4b92f45e63826a21dff1c2e76f169";
     private static final String TENANT_DOMAIN = "carbon.super";
+    private static final String QUERY_PARAMS = "client_id=MY_ACCOUNT&commonAuthCallerPath=%2Foauth2%2Fauthorize&forceAuth=false&passiveAuth=false&redirect_uri=https%3A%2F%2Fmyaccount.asg.io%2Fapp&t=testDomain&sp=application";
+    private static final String QUERY_PARAMS_WITHOUT_SP_T = "client_id=MY_ACCOUNT&commonAuthCallerPath=%2Foauth2%2Fauthorize&forceAuth=false&passiveAuth=false&redirect_uri=https%3A%2F%2Fmyaccount.asg.io%2Fapp";
```

Comment:
Let's use `localhost` for hostname

Reply:
Addressed with https://github.com/wso2-extensions/identity-outbound-auth-backup-code/pull/29/commits/fbf05731dfe2b29545ee7d2f61b0a385e6fe0ef5